File: 백준/Bronze/28074. 모비스/모비스.cs
string[] MOBIS = { "M", "O", "B", "I", "S" };
string word = Console.ReadLine();
bool result = true;

foreach (string letter in MOBIS)
{
    if (!word.Contains(letter))
    {
        result = false;
        break;
    }
}

if (result)
{
    Console.WriteLine("YES");
}
else
{
    Console.WriteLine("NO");
}

File: 백준/Bronze/28074. 모비스/모비스.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        String[] MOBIS = {"M", "O", "B", "I", "S"};
        Scanner scanner = new Scanner(System.in);
        String word = scanner.nextLine();
        boolean result = true;

        for (String letter : MOBIS) {
            if (!word.contains(letter)) {
                result = false;
                break;
            }
        }

        if (result) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
    }
}
